recipes/mrcpp/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set name = 'mrcpp' %}
+{% set version = "1.3.2" %}
+{% set sha256 = "61ffdfa36af37168090ba9d85550ca4072eb11ebfe3613da32e9c462351c9813" %}
+{% set build = 0 %}
+
+# recipe-lint fails if mpi is undefined
+{% set mpi = mpi or 'nompi' %}
+{% if mpi == "nompi" %}
+# prioritize shared-memory variant via build number
+{% set build = build + 100 %}
+{% endif %}
+# NOTE This means that the OpenMP build will be highest priority
+
+{% if mpi != "nompi" %}
+{% set mpi_prefix = "mpi_" + mpi %}
+{% else %}
+{% set mpi_prefix = "nompi" %}
+{% endif %}
+# add build string so packages can depend on
+# mpi or nompi variants
+# dependencies:
+# `mrcpp * mpi_mpich_*` for mpich
+# `mrcpp * mpi_*` for any mpi
+# `mrcpp * nompi_*` for no mpi
+
+# mpi builds require the right mpi
+# non-mpi builds *do not* appear to require non-mpi builds
+# at least not always
+
+{% if mpi != 'nompi' %}
+{% set build_pin = mpi_prefix + '_*' %}
+{% else %}
+{% set build_pin = '' %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MRChemSoft/mrcpp/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  binary_relocation: true
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+  run_exports:
+    - {{ pin_subpackage('mrcpp', max_pin='x.x') }}
+  run_exports:
```

Comment:
I think there should only be one `run_exports` key

Reply:
You mean, unify under one section:
```yaml
  run_exports:
    - {{ pin_subpackage('mrcpp', max_pin='x.x') }}
    - {{ name }} * {{ mpi_prefix }}_*
```
or is the second line redundant, given that `string` is present?

Reply:
Looking at the HDF5 `meta.yaml` (from which I adapted this one) I guess it should be:
```yaml
run_exports:
   - {{ pin_subpackage('mrcpp', max_pin='x.x') }} {{ build_pin }}
```

Reply:
I mean unify under one section

Reply:
I don't think it needs the build_pin

Reply:
Thanks :+1: I've consolidated `run_exports` now